apps/scan/frontend/src/components/ups_diagnostic_modal_button.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { DiagnosticOutcome } from '@votingworks/types';
+import { Button, Modal, P } from '@votingworks/ui';
+import * as api from '../api';
+
+export interface UpsDiagnosticModalProps {
+  setVisible: (visible: boolean) => void;
+}
+
+export function UpsDiagnosticModal(
```

Comment:
I see in https://github.com/votingworks/vxsuite/pull/7131 we add a version of this modal to `libs/ui`

Can we use that one here as well just to make sure copy remains consistent across the apps? Maybe that was already your plan and you're just planning to merge the VxCS PR and rebase this one on top or vice versa

Reply:
Ah I see now that the VxCS PR is built on top of this PR!

I'm sure you're already aware, but you can change the base branch of the VxCS PR to this PR's branch. Then when you merge this PR, the VxCS PR's base branch _should_ auto-update to be main

Reply:
Yeah sorry I meant to open that PR against this branch. For some reason I always expect github to open a PR for B against A when B branches off A, even though it has no way of knowing that's the user's intent.